minikubek8s.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Install kubectl
+echo "----------------------------kubectl checking----------------------------"
+export kubectl=$(which kubectl)
+if [ -f "$kubectl" ]; then
+    echo "$kubectl already exist...................................."
+else
+    echo "kubectl does not exist...................................."
+    echo "Installing kubectl........................................"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    kubectl version
+fi
+
+# Install Brew
+echo "-----------------------------brew checking------------------------------"
+export brew=$(which brew)
```

Comment:
I think there are better ways to check if something is installed instead of creating an env variable with the output of which.
For example, you can run `brew --version` and then depending on the exit code you can decide.